src/main/java/com/linecorp/armeria/client/circuitbreaker/metrics/DropwizardMetricsCircuitBreakerMonitor.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.circuitbreaker.metrics;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.codahale.metrics.MetricRegistry;
+
+import com.linecorp.armeria.client.circuitbreaker.CircuitBreaker;
+import com.linecorp.armeria.client.circuitbreaker.CircuitBreakerListener;
+import com.linecorp.armeria.client.circuitbreaker.CircuitState;
+import com.linecorp.armeria.client.circuitbreaker.EventCount;
+
+/**
+ * A {@link CircuitBreakerListener} which monitoring {@link CircuitBreaker} status using dropwizard metrics.
+ * <p>
+ * This class will generate the following metrics.
+ *
+ * <table summary="">
+ * <tr><th>metric name</th><th>description</th></tr>
+ *
+ * <tr><td>{prefix}.{circuit-breaker-name}.transitions.closed</td>
+ * <td>The number of times that the circuit state has changed to `CLOSED`.</td></tr>
```

Comment:
Unfortunately, Javadoc has no idea about backticks. Could we use `<code/>` instead?